spi/w5500_spi/socket.c

```diff
@@ -0,0 +1,931 @@
+//*****************************************************************************
+//
+//! \file socket.c
+//! \brief SOCKET APIs Implements file.
+//! \details SOCKET APIs like as Berkeley Socket APIs. 
+//! \version 1.0.3
+//! \date 2013/10/21
+//! \par  Revision history
+//!       <2015/02/05> Notice
```

Comment:
This is a frozen version. Instead of inserting this static version should be use the external library directly?